apps/mobile/src/screens/get-location/get-location.screen.tsx

```diff
@@ -25,12 +24,17 @@ const errorAlert = (msg: string, onPress: () => void, text?: string) =>
   ]);
 
 const GetLocation = () => {
-  const auth = useAuth();
-  const { getLocation, address, isLoading, warning } = useGeolocation();
+  const { getLocation, address, isLoading, warning, coords } = useGeolocation();
   const { t } = useLocale();
-  const { navigate } = useNavigation();
+  const { params } = useRoute();
 
-  const onConfirmLocation = () => navigate(Routes.Home as never);
+  const onConfirmLocation = () => {
+    const {
+      userInfo: { name, email, password },
+    } = params;
+
+    signUp({ name, email, password, coords });
```

Comment:
Please don't call the service function directly. Instead, instantiate [`useMutation`](https://tanstack.com/query/v4/docs/react/guides/mutations) and pass the service to it.

Reply:
Since we haven't finished https://github.com/animavita/animavita/issues/200 yet, please redirect the user to the login screen after the sign-up request succeeds.